catroid/src/main/java/org/catrobat/catroid/ui/adapter/BrickAdapter.java

```diff
@@ -1138,6 +1144,22 @@ private boolean brickHasAFormula(Brick brick) {
 		return (brick instanceof FormulaBrick || multiFormulaValid);
 	}
 
+	private boolean isBrickWithoutDescription(Brick brick) {
+		String name = brick.getClass().getSimpleName();
+		if (name.equals(IfLogicElseBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(IfLogicEndBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(IfThenLogicEndBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(LoopEndlessBrick.class.getSimpleName())) {
+			return true;
+		} else if (name.equals(LoopEndBrick.class.getSimpleName())) {
+			return true;
+		}
+		return false;
+	}
+
```

Comment:
Thanks for the update!
No need to revert the commit. Whenever you find the time, just create a PR with the suggested changes. :smile_cat: